src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2005, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (C) 2022 THL A29 Limited, a Tencent company. All rights reserved.
```

Comment:
No sure if we should add this new copyright line for Tencent.

But for hotspot changes, we usually just update the copyright year from 2021 to 2022 if the original copyright is from Oracle.

Reply:
Thank you for catching this.  Updated.